src/server/game/Anticheat/AnticheatMgr.cpp

```diff
@@ -230,6 +230,16 @@ void AnticheatMgr::ZAxisHackDetection(Player* player, MovementInfo movementInfo)
    if (m_Players[key].GetLastMovementInfo().HasMovementFlag(MOVEMENTFLAG_WATERWALKING) && movementInfo.HasMovementFlag(MOVEMENTFLAG_WATERWALKING))
        return;
 
+   switch (player->GetAreaId())
+   {
+   case 4281: //Acherus: The Ebon Hold
+   case 4342: //Acherus: The Ebon Hold
+       return;
+       break;
```

Comment:
break is not needed after return in a switch

Reply:
currently i am running a test which hopefully eliminates the ebon hold as a exception area. At the time i was just adding it in as consistency with code style.